indico/modules/events/timetable/forms.py

```diff
@@ -193,8 +184,7 @@ def data_for_format(self):
         if not self.advanced.data:
             fields = ('visible_entries',)
         else:
-            fields = set(get_form_field_names(TimetablePDFExportForm)) - self._pdf_options_fields - {'csrf_token',
-                                                                                                     'advanced'}
+            fields = set(get_form_field_names(TimetablePDFExportForm)) - {'csrf_token', 'advanced'}
         data = {}
```

Comment:
`data_for_format` is no longer used. However, right now we lose some functionality: The `advanced` toggle is now purely client-side, but maybe we should keep ignoring all the "advanced" form values when it's false?

Also, are `showSpeakerAffiliation` and `showSpeakerTitle` even used in legacy "simple" mode? If not we should probably hide them in simple mode...

Reply:
Correct, these two flags do not do anything in simple mode. I have now simply removed the distinction between simple and advanced mode. We have all the checkboxes with default values displayed now.